simiyu1/JavaDevsNairobi
# 155931587 create presenter to access api

#### What does this PR do?
 - Use githubpresenter class to consume the githubAPI
#### Description of Task to be completed?
 - Create a presenter class
 - Create service to leverage the githubAPI class
 - make live calls to the githubAPI and log results
 
#### How should this be manually tested?
 - On navigation to the presenter directory and you should see the[ GithubPresenter.java](https://github.com/simiyu1/JavaDevsNairobi/blob/ft-access-the-api-155931587/app/src/main/java/com/example/simiyunabende/javadevs/presenter/GithubPresenter.java) file
 - The above file should make calls to the githubAPI service

#### Any background context you want to provide?
 - Given I am generating GithubPresenter Class; then I should ensure that I construct a service leveraging the GithubApi interface.

#### What are the relevant pivotal tracker stories?
 - [#155931587](https://www.pivotaltracker.com/story/show/155931587)

#### Screenshots (if appropriate)
 - The presenter directory screenshot
<img width="255" alt="screen shot 2018-10-03 at 12 21 58" src="https://user-images.githubusercontent.com/17185082/46401923-048f5180-c707-11e8-9200-6200fb9be18b.png">
 - Screenshot of the logs showing fetched users
<img width="802" alt="screen shot 2018-10-03 at 12 21 39" src="https://user-images.githubusercontent.com/17185082/46402021-47e9c000-c707-11e8-9189-5dfc0f2dcdda.png">
